plugin/plugins/tk_window/__init__.py

```
@@ -101,3 +102,19 @@ def on_start(self, **_):
+        id="on_shutdown",
+        name="On Plugin Shutdown",
+        description="Run when plugin is stopped",
+        kind="hook",
+        auto_start=False,  # 不自动启动，插件停止时手动触发
+    )
+    def on_shutdown(self, **_):
+        # 在这里执行一些资源清理、状态保存等操作
+        self.ctx.logger.info("[tkWindow] plugin shutting down")
+        with self._lock:
+            if self._root is not None:
+                self._root.destroy()  # 如果窗口正在运行，销毁它
```

Ruff (0.14.7)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)